tests/test_mirror.py

```diff
@@ -13,9 +13,19 @@ def get_acl_table_id(self, dvs):
         assert dvs.asicdb.default_acl_table in keys
 
         acl_tables = [k for k in keys if k not in dvs.asicdb.default_acl_table]
-        assert len(acl_tables) == 1
+        assert len(acl_tables) >= 1
```

Comment:
what is the exact number of the ACL tables?

Reply:
DTel adds 2 additional ACL tables in addition to the default one